### PR TITLE
test(stdlib/testing): add tests for empty window nulls and fills

### PR DESCRIPTION
**Pull request by aanthony1243**
### Done checklist
- [X] Test cases written

### CODE REVIEW

File: stdlib/universe/universe.flux [hunk 1]
```diff
@@ -85,7 +85,7 @@ pearsonr = (x,y,on) => cov(x:x, y:y, on:on, pearsonr:true)
 // and then undo the windowing to produce an output table for every input table.
 aggregateWindow = (every, fn, columns=["_value"], timeSrc="_stop",timeDst="_time", tables=<-) =>
     tables
-        |> window(every:every)
+        |> window(every:every, createEmpty: true)
```
Comment (nathanielc) [inline stdlib/universe/universe.flux]: Did we want to make this configurable? Meaning add the createEmpty parameter to the aggregateWindow function and default it to `true`?
